etc/DFLMessages_en_GB.dic

```diff
@@ -37,16 +37,26 @@ RefUnsupportedTag             =     Settings ref %1% uses a unrecognized tag %2%
 
 //------------------ Main ---------------------------
 NetworkSlackNodeNotFound      =     Network slack node requested but not found in network input file %1%
-InputsInfo                    =     Processing network input file %1% and config file %2%
+SteadyStateInfo               =     Processing network input file %1% and config file %2% for steady-state calculation
+SecurityAnalysisInfo          =     Processing network input file %1%, contingencies %2% and config file %3% for security analysis
 ExportInfo                    =     Exporting outputs files for %1%
 SlackNode                     =     Slack node of id %1% found with origin %2% (0=file, 1=algorithm)
 ConnexityError                =     Slack node of id %1% not present in main connex component
 ConnexityErrorReCompute       =     Slack node of id %1% not present in main connex component: compute slack node only in main connex component
 SimulateInfo                  =     Initializing %1% simulation
 NetworkFileNotFound           =     Network file %1% does not exist
 NetworkHasNoRegulatingGenerator         =     Network file %1% has no generator regulating voltage in the main connex component
+ContingenciesFileNotFound     =     Contingencies file %1% was given but does not exist
 ContextProcessError           =     Processing of the network %1% failed
 SimulationEnded               =     Simulation %1% ended successfully (wall-time: %2%s)
 DFLEnded                      =     DynaFlowLauncher %1% ended successfully (wall-time: %2%s)
 InitEnd                       =     End of initialization (wall-time: %1%s)
 FilesEnd                      =     End of files generation (wall-time: %1%s)
+
+//-------------- Contingencies -----------------------
```

Comment:
Sections in dictionnary correspond to internal libraries (inputs, algo, outputs and main executable). The contingencies shall be located in the section where they are used

Reply:
Rearranged dictionary entries related to contingencies, moving each entry to the section where it is used